rainbow-me/rainbowkit
# fix: `<ConnectButton.Custom />` not displaying balance

## Changes
- `<ConnectButton.Custom />` doesn't give balance back due to our `showBalance` context API blocking it. It's now forced to show balance. 
- With `<ConnectButton />` we wait until context API gets updated and then decide whether we show the balance or not. This way we don't show balance right away, but rather wait until we get some value from the context API.

## What to test
1. Do not test `<ConnectButton />` and `<ConnectButton.Custom />` at the same time since it'll force call rpc balance if you use the custom one.
2. Make sure when testing `<ConnectButton.Custom />` it should show balance no matter what. Look at network tab and see that the rpc is being called.
3. Make sure when testing with `<ConnectButton />` it should work as is right now with different responsive screen sizes.

_We don't want to expose another prop to our `<ConnectButton.Custom />` so we'll try to use context API as much as possible here 👍_
